syntax = "proto3";

package story;

service StoryService {
  rpc GenerateStory (StoryRequest) returns (StoryResponse);
}

message StoryRequest {
  string source = 1;
  string theme = 2;
}

message StoryResponse {
  string story = 1;
  string error = 2;
}
